tests/bits/prm/parameter_handler_backslash_04.prm

```diff
@@ -0,0 +1,6 @@
+#Listing of Parameters
+#---------------------
+subsection Testing
+  set Function = a,\ b,
+                 c
```

Comment:
this should actually yield an error, but it doesn't currently because you don't check *where* the backslash is -- as long as there *is* a backslash somewhere in the line, you treat it as a continued line, but that seems wrong to me.